source/MaterialXView/Viewer.cpp

```diff
@@ -78,13 +78,18 @@ mx::DocumentPtr loadLibraries(const mx::StringVec& libraryFolders, const mx::Fil
         mx::FilePath path = searchPath.find(libraryFolder);
         mx::FilePathVec filenames = path.getFilesInDirectory("mtlx");
 
+        mx::CopyOptions copyOptions;
```

Comment:
For the Substance files we still need this as reads can have xincludes which conflict, and imports can still also conflict.